docker-local/setup.sh

```
@@ -52,15 +52,18 @@ docker run --rm \
 
     # Scoped signing key that auth-service uses to sign user JWTs. The role
     # template mirrors the grants auth-service used to inline per JWT, keyed
-    # off the account:<account> tag every user JWT now carries.
+    # off the account:<account> tag every user JWT now carries. The inbox is
+    # per-account and subscribe-only: a client holds no standing publish grant
+    # on any inbox. The one-shot allow-pub-response lets a client publish a
+    # reply only to a request it itself received, so no client can read
+    # another's replies or forge another client's reply.
```

Shellcheck (0.11.0)

[warning] This apostrophe terminated the single quoted string!

(SC1011)